tests/docarray/test_add.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from docarray import Document, DocumentArray
+
+
+def test_add():
+    annlite_doc = DocumentArray(
```

Comment:
`annlite_doc` should be created as `pytest.fixture` in `conftext.py`. 

And the test docs should also use fixture defined in `conftext.py`